src/execution_tree/primitives/slice.cpp

```diff
@@ -275,16 +280,85 @@ namespace phylanx { namespace execution_tree
                     name, codename));
         }
 
-        if (data.has_annotation() || value.has_annotation())
+//        if (data.has_annotation() || value.has_annotation())
```

Comment:
I guess we can remove these commented lines

Reply:
Yes. Will delete it.